chapters/en/slides/module5/module5_03.Rmd

```diff
@@ -0,0 +1,316 @@
+
+However,
+we cannot see exactly which values the data points during the first half of the year has.
+Are they close to zero? Or 10,000? We can't know from looking at this plot.
+
+---
+
+## We can zoom in to highlight a region of an axes by filtering the data
+
+```{python}
+title_text = 'Wikipedia donations peak during Christmas'
+alt.Chart(donations[donations['date'].between("2020", "2020-07")], title=title_text).mark_circle().encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day'))
+```
+
+Notes:
+Zooming in on an axis is useful
+when we want to highlight a certain region.
+The easiest way to achieve this
+is usually to filter the data we pass to `Chart` as in this example.
+
+---
+
+## We can zoom in to highlight a region of an axes
+
+```{python}
+title_text = 'Wikipedia donations peak during Christmas'
+alt.Chart(donations[donations['date'].between("2020", "2020-07")], title=title_text).mark_circle(clip=True).encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day', scale=alt.Scale(domain=[0, 100_000])))
+```
+
+Notes:
+It is not always practical to filter the data,
+so it is good to know how to change the axis range directly in Altair.
+For this,
+we could modify the domain of the axis' scale.
+
+When we used `alt.Axis` previously,
+the changes we made were cosmetic,
+such as tick numbers,
+label formatting etc.
+
+Now that were are modifying the range of the actual displayed values,
+we need to use `alt.Scale` instead.
+It also works for colors and sizes in a legend,
+just like `alt.Axis`.
+
+Finally we need to set `clip=True` in the axis,
+to be explicit about that we are excluding data.
+Without it,
+we would still see the circles extending beyond the range of the chart.
+
+This ensures that we are intentional about our data exclusion,
+and it is something we should think twice about because it can skew the perception of the data.
+For example,
+as we saw in the previous slide deck,
+we should not do in to a bar chart that shows counts or proportions.
+
+After zooming in on the y-axis,
+we can more clearly see the variation in the lower value data points.
+
+---
+
+## Axes can be anchored to the data range instead of to zero without explicitly setting the domain
+
+```{python}
+alt.Chart(donations[donations['sum'] > 2.5e6]).mark_circle().encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day'))
+```
+
+Notes:
+You might have noticed from previous modules that Altair often includes 0 in the axis,
+even if our data does not have any values close to 0.
+The reason for this is to enforce good habits 
+and not create for example bar plot that are cut off
+which makes difference appear much bigger than they actually are.
+
+However,
+there are also good reasons for setting the range based on the extent of the data,
+e.g. do be able to show the variation fully.
```

Comment:
I think a date/time example might be a better one. Like what if you were plotting time vs something, but you only had measurements from the evening. There would not be much sense in having half-three quarters of the plot empty...